docs/welcome.md

```diff
@@ -1,54 +1,59 @@
 ---
 id: welcome
-title: Get Started
-description: Master Amplication with in-depth developer guides, API details, and tutorials.
-sidebar_label: Introduction
+title: Amplication Documentation
+description: Learn how Amplication helps teams build standardized backend services while enforcing best practices and preventing technical debt.
+sidebar_label: Overview
 slug: /
 ---
 
-# Get Started with Amplication
+# Amplication Documentation
 
-Welcome to Amplication's developer documentation!
+Amplication helps you build, maintain, and scale backend services that align with your organization's best practices, supporting you throughout the entire development process.
+Our platform combines live templates, AI-powered code generation, and technical debt management features to help teams create production-ready Node.js and .NET backend services while enforcing organizational standards.
 
-Amplication is an open-source, AI-powered code generation solution for rapid .NET and Node.js development. Unlike other AI code generation tools, Amplication is the only one that guarantees production-ready code generation for all your backend needs.
-
-## Quick Start
+## Quickstart
 
 [Create your first service](/first-service/) with Amplication's AI, Jovu, and generate a .NET or Node.js _production-ready codebase in minutes_.
 
-## Customize Your Generated Service
+## AI-powered Backend Generation
```

Comment:
Since our advantages reside mostly in the standards and tech debts, I think we should push the "AI-powered" section to be after the 'tech debt" section.
WDYT?